src/api/v1/serializers/doughs-serializer.js

```diff
@@ -19,6 +19,20 @@ const doughResourceUrl = resourcePathLink(apiBaseUrl, doughResourcePath);
  */
 const transformSpecialInstructions = (dough) => {
   dough.specialInstructions = dough.specialInstructions || '';
+
+  const intFields = [
+    'gramsFlour',
+    'gramsWater',
+    'waterTemp',
+    'gramsYeast',
+    'gramsOliveOil',
+    'bulkFermentTime',
+    'proofTime',
+  ];
```

Comment:
What about `gramsSalt` and `gramsSugar`? https://github.com/osu-mist/pizza-api/blob/cb99cca40ffe2d5e76f3cdd6236b597c2faa3132/openapi.yaml#L848-L856

Reply:
Whoops, good catch